elements/howto-tabs/howto-tabs.js

```diff
@@ -57,10 +70,15 @@
       this.attachShadow({mode: 'open'});
       // Import the shared template to create the slots for tabs and panels.
       this.shadowRoot.appendChild(
-        document.importNode(shadowDOMTemplate.content, true)
+        document.importNode(template.content, true)
```

Comment:
I was asking myself the same question. Not sure why I used it in the first place. I feel like it had a reason, but I don’t remember :-/